File: Desafio 1/js_marcelofigueira/desafio1.js
var desafio1 = function(){
var santa = 'Santa';
var rem = 'rem';
    for (var i = 1; i <= 200; i++){
        if (i % 5 == 0 && i % 6 == 0 ){
            console.log(i + ' ' + santa + rem);
        } else if (i % 6 == 0){
            console.log(i + ' ' + rem);
        } else if(i % 5 == 0){
            console.log(i + ' ' + santa );
        } else {
            console.log(i);
        }
    }  
}

desafio1();

File: Desafio 1/Ruby(aryrabelo)/desafio1.rb
class Desafio1
  def self.perform
    (1..200).to_a.map do |num|
      number(num)
    end
  end

  def self.number(n)
    "#{n} #{show_santa?(n)}#{show_rem?(n)}".strip
  end

  def self.show_santa?(n)
    "Santa" if n % 5 == 0
  end

  def self.show_rem?(n)
    "rem" if n % 6 == 0
  end
end

puts Desafio1.perform